package.json

```diff
@@ -293,34 +402,54 @@
             "view/item/context": [
                 {
                     "$comment": "============= Treeview commands =============",
-                    "command": "azurerm-vscode-tools.sortTemplate",
-                    "when": "azurerm-vscode-tools.template-outline.active == true",
-                    "group": "arm-template"
-                },
-                {
                     "command": "azurerm-vscode-tools.sortFunctions",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == functions",
-                    "group": "arm-template"
+                    "group": "inline"
                 },
                 {
                     "command": "azurerm-vscode-tools.sortOutputs",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == outputs",
-                    "group": "arm-template"
+                    "group": "inline"
                 },
                 {
                     "command": "azurerm-vscode-tools.sortParameters",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == parameters",
-                    "group": "arm-template"
+                    "group": "inline"
                 },
                 {
                     "command": "azurerm-vscode-tools.sortResources",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == resources",
-                    "group": "arm-template"
+                    "group": "inline"
                 },
                 {
                     "command": "azurerm-vscode-tools.sortVariables",
                     "when": "azurerm-vscode-tools.template-outline.active == true && viewItem == variables",
-                    "group": "arm-template"
+                    "group": "inline"
```

Comment:
I think we should have both the inline versions *and* the right-click versions, mostly for accessibility reasons - it's easier for those using just a keyboard and no mouse to activate the right-click versions.

Reply:
Fixed.